cmd/gateway-main.go

```diff
@@ -215,6 +215,8 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 		logger.FatalIf(err, "Unable to initialize gateway backend")
 	}
 
+	logger.FatalIf(globalPolicySys.Init(newObject), "Unable to initialize policy system")
```

Comment:
@harshavardhana, why is this required for gateway? globalPolicySys creates a new policy system already on line 177. Policy cache can be filled up as requests come in for the bucket instead of pre populating policy cache for all the buckets. Unsure of what this PR is fixing.

Reply:
Without Init policies are not loaded.. and they never get loaded.